src/grt/src/GlobalRouter.tcl

```diff
@@ -193,6 +193,7 @@ proc set_global_routing_random { args } {
 sta::define_cmd_args "global_route" {[-guide_file out_file] \
                                   [-congestion_iterations iterations] \
                                   [-congestion_report_file file_name] \
+                                  [-drc_report_iter_step steps] \
```

Comment:
I'm not sure if this is the best name for this option. Maybe "report_partial_congestion iteration_interval", or something like that.

Also, please add the description of this new option to the README.md file.